float64 avg
float64 std
float64 min
float64 max
